src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java

```diff
@@ -278,6 +279,29 @@ private boolean isFieldSetForSelectedEntry(String fieldname) {
         }
     }
 
+    private boolean areAllFieldsSetForSelectedEntry(List<String> fieldnames) {
+        if (panel.getMainTable().getSelectedRowCount() == 1) {
+            BibEntry entry = panel.getMainTable().getSelected().get(0);
+            return entry.getFieldNames().containsAll(fieldnames);
+        } else {
+            return false;
+        }
+    }
+
+    private boolean isAnyFieldSetForSelectedEntry(List<String> fieldnames) {
+        if (panel.getMainTable().getSelectedRowCount() == 1) {
+            BibEntry entry = panel.getMainTable().getSelected().get(0);
+            for (String fieldname : fieldnames) {
```

Comment:
Simple one liner: `! Collections.disjoint(list1, list2); `
From the apidoc:  disjoint (a, b) Returns true if the two specified collections have no elements in common.